src/redux/bookCatego/categSlice.js

```diff
@@ -0,0 +1,16 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const myCateg = 'Categories';
+const initialState = {
+  bookName: myCateg,
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    statusCheck: () => 'Under construction',
+  },
+});
```

Comment:
- [ ] Well done 👏🏼 implementing the category slice, however, if you check the project requirement: 
![image](https://user-images.githubusercontent.com/68030297/236882132-dd323fb9-2d51-4622-b1bd-4c5f025a7687.png)

- So basically you need to send action payload `data` with the action creator to check if the data is "Under construction" Then the reducers will return "Under construction" otherwise return the state, follow the below example to get a better idea:

```javascript
reducers: {
    statusCheck: (state, action) => {
        if (action.payload === "Under construction") {
            return "Under construction"
         }
         // Otherwise
         return state.categories
}
```
This is just an example that you can inspire from and match it with your code 👍🏼